composer.json

```diff
@@ -24,7 +24,7 @@
   "require": {
     "typo3/cms-core": "^11.5",
     "typo3/cms-dashboard": "^11.5",
-	  "php": "^7.4 || ^8.0 || ^8.1",
+	  "php": "^7.4 || ^8.0 || ^8.1 || ^8.2",
```

Comment:
FYI, `^8.0` already includes `8.1` and any other later `8.x`, so this is redundant.

What you really want is to say its compatible with "8.0, 8.1, 8.2", but once 8.3 comes out you cannot be sure, so safer would be:

`"php": ">=8.0, <=8.2"`